client/src/components/button/spec/Button.spec.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { mount } from "enzyme";
+
+import Button from "..";
+
+describe("Button", () => {
+    it("рендерится корректно", () => {
+        const wrapper = mount(
+            <Button
+                type="button"
+                style="primary"
+                stretch={false}
+                clickHandler={() => {}}
```

Comment:
Для создания моков в `jest` можно использовать `jest.fn`

```suggestion
                clickHandler={jest.fn()}
```